connectors/src/connectors/github/lib/github_api.ts

```diff
@@ -97,18 +99,37 @@ export async function getGithubAppPrivateKey(): Promise<string> {
 export async function installationIdFromConnectionId(
   connectionId: string
 ): Promise<string | null> {
-  // TODO(spolu): GITHUB_MIGRATION dual mode will pull the refresh token to get the installation id
-  // from the scrubbed_raw_json.
-  const octokit = await getOctokit(connectionId);
+  if (isDualUseOAuthConnectionId(connectionId)) {
```

Comment:
nit: maybe a small comment to explain what "dual use oauth" means could help a bit

Reply:
to be fair I left a comment with the definition of that function :)

Reply:
Fair fair